.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Setup and Test
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```

Comment:
**Question:** ArcGIS Pro is currently on 3.6.10. Even though we can't install arcpy here, do we want to keep this version consistent with what most people will have in their conda environments by default?

Reply:
Good question. Because this template is not specifically designed to be used with arcpy and any version of python _could be installed_, I'd rather not limit this to the pro conda environment. 

What we should do is test on a few different versions of python including LTS (if they use that term) and 3.6.

If arcpy is going to be involved with the template, some extra steps have to be taken, like downgrading pytest.

Reply:
I added new issues for arcpy integration and testing multiple python versions. Both of which I think would be great improvements.